test/load/README.md

```diff
@@ -0,0 +1,30 @@
+# Load tests
+
+See [locustfile.py](locustfile.py) for descriptions of task sets implemented using [locust](https://github.com/locustio/locust) framework.
+
+## Prerequisites
+
+ * Docker
+ * Docker Compose
+ * Python 3
+
+## Build steps
+
+1. Create a Docker image with Python, locust, and gRPC.
+```sh
+docker build -t iroha-locust .
+```
+
+2. Copy [irohalib.py](https://github.com/hyperledger/iroha/blob/master/example/python/irohalib.py) and [ed25519.py](https://github.com/hyperledger/iroha/blob/master/example/python/ed25519.py) and follow the preparation steps in [libiroha.md](https://github.com/hyperledger/iroha/blob/master/example/python/irohalib.md).
```

Comment:
irohalib in master do not has the latest version (not all commands are supported). should we still use it?